### PR TITLE
Restrict address fields to 50 characters

### PR DESCRIPTION
**Pull request by stevenwilkin**

### CODE REVIEW

File: app/models/appointment_summary.rb [hunk 1]
```diff
@@ -32,7 +32,8 @@ def postcode=(str)
   has_many :appointment_summaries_batches, dependent: :destroy
 
   validates :title, presence: true, inclusion: { in: TITLES, allow_blank: true }
-  validates :last_name, presence: true
+  validates :first_name, length: { maximum: 50 }
+  validates :last_name, presence: true, length: { maximum: 50 }
```
Comment (andrewgarner) [inline app/models/appointment_summary.rb]: I think the desired behaviour is that the combined `title`, `first_name` and `last_name` can't be greater than 50 characters. This is so the full name can be seen when shown through a C5 envelope window.

File: app/models/appointment_summary.rb [hunk 2]
```diff
@@ -53,9 +54,12 @@ def postcode=(str)
...
 
-  validates :address_line_1, presence: true
-  validates :town, presence: true
-  validates :postcode, presence: true, postcode: true
+  validates :address_line_1, presence: true, length: { maximum: 50 }
+  validates :address_line_2, length: { maximum: 50 }
+  validates :address_line_3, length: { maximum: 50 }
+  validates :town, presence: true, length: { maximum: 50 }
+  validates :county, length: { maximum: 50 }
+  validates :postcode, presence: true, postcode: true, length: { maximum: 50 }
```
Comment (andrewgarner) [inline app/models/appointment_summary.rb]: We don't need to validate length of postcode. The postcode validation will handle this.

File: app/models/appointment_summary.rb (hunk 1 above)
Comment (stevenwilkin) [inline app/models/appointment_summary.rb]: Ok!

Comment (gjvis) [inline app/models/appointment_summary.rb]: Yeah, its `OutputDocument#attendee_name` that can't be more than 50. Or perhaps more specifically, no individual line of `OutputDocument#attendee_address`

File: app/models/appointment_summary.rb (hunk 2 above)
Comment (stevenwilkin) [inline app/models/appointment_summary.rb]: Good spot, copy'n'paste fail

**Comment by gjvis** [comment]
:+1:

**Pull request merged**